libtransmission/torrent.h

```diff
@@ -240,6 +241,8 @@ struct tr_torrent
     uint16_t idleLimitMinutes;
     tr_idlelimit idleLimitMode;
     bool finishedSeedingByIdle;
+
+    tr_ptrArray* labels;
```

Comment:
The reason we don't have a `tr_ptrArrayNew()` and `tr_ptrArrayFree()` anywhere in the rest of the code is that sizeof(tr_ptrArray) is so small that it's better to just aggregate it directly and avoid the extra malloc/free for the tr_ptrArray struct. So:

- consider using `tr_ptrArray labels;` here
- if we do that, we can remove `tr_ptrArrayNew()`
- and `tr_ptrArrayFree()`